XenModel/XenAPI/VM.cs

```diff
@@ -3798,6 +3798,20 @@ public virtual vm_power_state power_state
         private vm_power_state _power_state;
 
         /// <summary>
+        /// Call a XenAPI plugin on this vm

```

Comment:
Hi @stephen-turner 
Do you mean we can just copy VM.cs and Proxy.cs from build system?
But currently we do not have a release version which include these interface, do we still need to wait for a specific version?